tests/python/dashboard/test_dashboard.py

```diff
@@ -0,0 +1,82 @@
+import importlib
+
+from util import set_input_value, start_dashboard, wait_for_ready
+import time
+
+import pytest
+
+
+@pytest.mark.skipif(
+    importlib.util.find_spec("seleniumbase") is None, reason="seleniumbase is not available"
+)
+def test_simulation():
+    """
+    This test runs the FODO example on the dashboard and verifies
+    that the simulation has ran successfully.
+    """
+    from seleniumbase import SB
+
+    app_process = start_dashboard()
+    time.sleep(10)
+
+    try:
+        with SB() as sb:

```

Comment:
Trying if this prevents that the unit test on Linux/macOS opens a browser:
```suggestion
        with SB(headless=True) as sb:

```

Reply:
https://github.com/ECP-WarpX/impactx/pull/672/files#r1717652047